snapcraft/internal/lxd.py

```diff
@@ -29,7 +29,7 @@
 import yaml
 
 from snapcraft.internal.errors import SnapcraftEnvironmentError
-from snapcraft.internal import common
+from snapcraft.internal.common import format_snap_name, is_snap
```

Comment:
I prefer leaving this as `common`, personally, unless it's clear from the context where it's coming from (e.g. importing matchers from testtools in a test is clear). Otherwise I feel it decreases readability. @elopio, what are your thoughts?